.github/workflows/ci.yml

```diff
@@ -2,7 +2,7 @@ name: CI
 
 on:
   pull_request:
-    types: ['opened', 'edited', 'reopened', 'synchronize']
+    types: ["opened", "edited", "reopened", "synchronize"]
```

Comment:
At some point we should agree on a format, what does prettier say ?

Reply:
Sorry i used yaml format instead of prettier

Reply:
Indent is with prettier too ?

Reply:
I used prettier after your comment